phenotrex/transforms/annotation.py

```diff
@@ -127,15 +147,17 @@ def annotate_with_deepnog(
     protein_list: List[SeqRecord],
     database: str = 'eggNOG5',
     tax_level: int = 2,
+    confidence_threshold: float = None,
     verb: bool = True
 ) -> GenotypeRecord:
     """
-    Perform calling of EggNOG5 clusters on a list of SeqRecords belonging to a sample, using deepnog.
+    Perform calling of clusters on a list of SeqRecords belonging to a sample using deepnog.
```

Comment:
How about:
```suggestion
    Assign proteins belonging to a sample to orthologous groups using deepnog.
```
for consistency with nomenclature in the deepnog paper. Also I'd not describe the input data again, as it's anyway annotated in the function signature.